examples/usdConverter/usdConverter.cc

```diff
@@ -20,7 +20,7 @@
 #include <pxr/usd/usd/stage.h>
 
 #include "sdf/sdf.hh"
-#include "sdf_usd_parser/world.hh"
+#include "sdf/world.hh"
```

Comment:
I tried following the file structure for components in `ign-common`:
```
<component name>
    |__include
    |    |__<project name>
    |         |__component header files
    |__src
         |__component source files
```

This means that for the USD component, header files are in the path `usd/include/sdformat`, and when it's time to `#include` them in the component's source files, they have the same prefix as sdformat's header files (`sdf/`). This doesn't result in naming collisions _right now_ because the component header files are not upper case, but a lot of the USD component header files are essentially just a lower case name of the sdformat header files, which can make this code confusing to read (example: `world.hh` vs `World.hh`). Should I change the component header file structure somehow so that the `#include` path is unique (for example, something like `#include "sdf/usd/..."`)?